### ISSUE TITLE
Not installed package: detail window filled with random(?) documents

### ISSUE THREAD

**Issue by jmsoeiro** [amaxwell/tlutility#90]
tlu: 1.36
TexLive: 2020
Mac OSX Catalina 10.15.4

When clicking on a "Not installed package", the detail window gives the correct description of the package, but the wrong set of documents.
Detected with "bookshelf" package, which sometimes shows "booklet" documentation, others "worksheet" or "bookest" docs.
Maybe it's a corrupt database, and not tlu's fault?  (I changed mirror, but the behavior is maintained)
<img width="498" alt="Screenshot 2020-05-30 at 17 13 47" src="https://user-images.githubusercontent.com/13068758/83333551-fc4f3700-a298-11ea-9458-f6b8907f1745.png">

**Comment by amaxwell** [comment]
Please try https://github.com/amaxwell/tlutility/releases/download/1.37b2/TeX.Live.Utility.app-1.37b2.dmg and let me know if the problem is fixed. Texdoc returns junk results when a package isn't installed, and I thought I'd fixed this before, but apparently not. There may be some other edge cases to look at (e.g., BasicTeX, which doesn't have documentation).

**Comment by jmsoeiro** [comment]
Sorry, but no. 

Still have strange documents, somewhat related: 
- booklet instead of bookshelf,
- tikz-layers <->  tiklz-lakes;
- eledmac <-> pwebmac

Anyway, it's not a dramatic issue, because you can always get the docs at CTAN...

To test, installed bookshelf, and the right docs were produced.
I tried to remove bookshelf again, but an error occurred.

I attach the log:

2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment initialize][48174]	Welcome to TeX Live Utility 1.37b2, running under Mac OS X Version 10.15.4 (Build 19E287) with 8/8 processors active and 8.0 GB physical memory.
2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment _checkProcessUmask][48174]	Process umask = 022
2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment updateEnvironment][48174]	Setting up a new environment for /usr/local/texlive/2020…
2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment updatePathEnvironment][48174]	Using PATH = "(
    "/Library/TeX/texbin",
    "/usr/bin",
    "/bin",
    "/usr/sbin",
    "/sbin"
)"
2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment _checkSystemPythonMajorVersion:minorVersion:][48174]	Using python at '/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python'
2020-05-31 15:03:25 +0000 Notice +[TLMEnvironment _checkSystemPythonMajorVersion:minorVersion:][48174]	Python version is 2.7.16 (default, Feb 29 2020, 01:55:37) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.29.20) (-macos10.15-objc-
2020-05-31 15:03:26 +0000 Notice +[TLMEnvironment updateEnvironment][48174]	Nothing to update for /usr/local/texlive/2020
2020-05-31 15:03:26 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Checking the repository version.  Please be patient.
2020-05-31 15:03:26 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Downloading at least 2048 bytes of tlpdb for a version check…
2020-05-31 15:03:26 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Downloaded 65536 bytes of tlpdb for version check
2020-05-31 15:03:26 +0000 Notice -[TLMEnvironment initWithInstallDirectory:][48174]	Looks like you're using TeX Live 2020
2020-05-31 15:03:26 +0000 Notice -[TLMEnvironment _checkForRootPrivileges][48174]	Root permission required for installation at /usr/local/texlive/2020
2020-05-31 15:03:26 +0000 Notice +[TLMEnvironment _logEnvironment][48174]	Current environment from /usr/bin/env:
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0
CC_FOR_BUILD=no_compiler_found
COMMAND_MODE=unix2003
DISPLAY=/private/tmp/com.apple.launchd.1M7XmpWXKU/org.macosforge.xquartz:0
HOME=/Users/josesoeirocarvalho
LOGNAME=josesoeirocarvalho
PATH=/Library/TeX/texbin:/usr/bin:/bin:/usr/sbin:/sbin
SHELL=/bin/zsh
SQLITE_EXEMPT_PATH_FROM_VNODE_GUARDS=/Users/josesoeirocarvalho/Library/WebKit/Databases
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.luB84NkOs5/Listeners
TMPDIR=/var/folders/__/26q5tl3x7lg5pmt_613qx6r40000gn/T/
USER=josesoeirocarvalho
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
2020-05-31 15:03:26 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Reloading local tlpdb from file:///usr/local/texlive/2020/tlpkg/texlive.tlpdb
2020-05-31 15:03:28 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Took 2.47 seconds to reload local tlpdb
2020-05-31 15:03:28 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Checking the repository version.  Please be patient.
2020-05-31 15:03:28 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Downloading at least 2048 bytes of tlpdb for a version check…
2020-05-31 15:03:28 +0000 Notice -[TLMDatabase _downloadDatabaseHead][48174]	Downloaded 2641 bytes of tlpdb for version check
2020-05-31 15:03:28 +0000 Notice -[TLMEnvironment _getValidServerURL:repositoryYear:fromURL:][48174]	Repository version appears to be 2020; a young TeX Live
2020-05-31 15:03:28 +0000 Notice -[TLMMainWindowController _refreshUpdatedPackageListFromLocation:][48174]	Refreshing list of updated packages…
2020-05-31 15:03:31 +0000 Notice -[TLMMainWindowController _paperSizeCheckTerminated:][48174]	System paper size = a4, pdftex paper size = a4
2020-05-31 15:03:32 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --machine-readable --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet update --list --all`
2020-05-31 15:03:32 +0000 Notice -[TLMOperation main][48174]	start load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
finish load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
2020-05-31 15:03:34 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --machine-readable --repository http://www.preining.info/tlgpg/ update --list --all`
2020-05-31 15:03:34 +0000 Notice -[TLMOperation main][48174]	start load http://www.preining.info/tlgpg/
finish load http://www.preining.info/tlgpg/
2020-05-31 15:03:34 +0000 Notice -[TLMMainWindowController _handleListGPGUpdatesFinishedNotification:][48174]	Found 0 tlgpg packages to update from http://www.preining.info/tlgpg/
2020-05-31 15:03:50 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Reloading local tlpdb from file:///usr/local/texlive/2020/tlpkg/texlive.tlpdb
2020-05-31 15:03:53 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Took 2.49 seconds to reload local tlpdb
2020-05-31 15:03:53 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	bookshelf not found in database; reverting to `tlmgr show`.
2020-05-31 15:03:55 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list bookshelf`
2020-05-31 15:04:02 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	pwebmac not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:03 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	pwebmac not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:03 +0000 Notice -[TLMOperation main][48174]	Cancelled `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list pwebmac`
2020-05-31 15:04:06 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list pwebmac`
2020-05-31 15:04:15 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	biblatex-vancouver not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:15 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	biblatex-vancouver not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:16 +0000 Notice -[TLMOperation main][48174]	Cancelled `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list biblatex-vancouver`
2020-05-31 15:04:19 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list biblatex-vancouver`
2020-05-31 15:04:20 +0000 Notice -[TLMInfoOperation main][48174]	Unable to find documentation for biblatex-vancouver
2020-05-31 15:04:23 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	utf8add not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:24 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	utf8add not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:24 +0000 Notice -[TLMOperation main][48174]	Cancelled `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list utf8add`
2020-05-31 15:04:27 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list utf8add`
2020-05-31 15:04:31 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	tile-graphic not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:34 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list tile-graphic`
2020-05-31 15:04:39 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	tikz-lake-fig not found in database; reverting to `tlmgr show`.
2020-05-31 15:04:42 +0000 Notice -[TLMOperation main][48174]	Successfully executed `/Library/TeX/texbin/tlmgr --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet show --list tikz-lake-fig`
2020-05-31 15:06:16 +0000 Notice -[TLMMainWindowController _refreshFullPackageListFromLocation:offline:][48174]	Refreshing list of all packages…
2020-05-31 15:06:21 +0000 Notice -[TLMLoadDatabaseOperation _dumpDatabaseAtURL:asPropertyList:][48174]	Dump error: finish load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet

2020-05-31 15:06:22 +0000 Notice -[TLMLoadDatabaseOperation main][48174]	Unable to read mirror from tlpdb property list with keys (
    packages
)
2020-05-31 15:06:26 +0000 Notice +[TLMDatabase packageNodesByMergingLocalWithMirror:][48174]	7158 packages in repository database, 4024 packages in local database
2020-05-31 15:10:38 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	pwebmac found in database; bypassing tlmgr.
2020-05-31 15:12:22 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	bookshelf found in database; bypassing tlmgr.
2020-05-31 15:12:29 +0000 Notice -[TLMMainWindowController _isCorrectDatabaseVersionAtURL:][48174]	Checking database version in case preferences have been changed…
2020-05-31 15:12:29 +0000 Notice -[TLMMainWindowController _installPackagesWithNames:reinstall:][48174]	Beginning install of (
    bookshelf
)
from http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
2020-05-31 15:12:29 +0000 Notice -[TLMAuthorizedOperation main][48174]	Checking code signature before running tlu_ipctask as root…
2020-05-31 15:12:29 +0000 Notice /usr/bin/codesign[48174]	/private/var/folders/__/26q5tl3x7lg5pmt_613qx6r40000gn/T/AppTranslocation/2BAEA08A-9B5B-4CE0-94C3-0909859DDC2B/d/TeX Live Utility.app/Contents/MacOS/tlu_ipctask: valid on disk
/private/var/folders/__/26q5tl3x7lg5pmt_613qx6r40000gn/T/AppTranslocation/2BAEA08A-9B5B-4CE0-94C3-0909859DDC2B/d/TeX Live Utility.app/Contents/MacOS/tlu_ipctask: satisfies its Designated Requirement
2020-05-31 15:12:29 +0000 Notice -[TLMAuthorizedOperation main][48174]	Signature was valid, okay to run tlu_ipctask
2020-05-31 15:12:41 +0000 Notice -[TLMAuthorizedOperation main][48174]	Invoking privileged task via AuthorizationExecuteWithPrivileges
2020-05-31 15:12:41 +0000 Notice tlu_ipctask[48950]	dropped privileges to user nobody
2020-05-31 15:12:42 +0000 Notice -[TLMAuthorizedOperation setWrapperPID:][48174]	tlu_ipctask checking in:  tlu_ipctask pid = 48950
2020-05-31 15:12:42 +0000 Notice -[TLMAuthorizedOperation setUnderlyingPID:][48174]	tlu_ipctask checking in: pid = 48951 (/Library/TeX/texbin/tlmgr --machine-readable --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet --persistent-downloads install bookshelf)
2020-05-31 15:12:42 +0000 Notice tlu_ipctask[48950]	tlu_ipctask: child HOME = '/var/root'
2020-05-31 15:12:42 +0000 Notice tlu_ipctask[48950]	tlu_ipctask: current HOME = '/var/empty'
2020-05-31 15:12:44 +0000 Warning tlu_ipctask[48950]	start load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
2020-05-31 15:12:46 +0000 Warning tlu_ipctask[48950]	finish load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
2020-05-31 15:12:47 +0000 Notice tlu_ipctask[48950]	location-url	http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet
2020-05-31 15:12:47 +0000 Notice tlu_ipctask[48950]	Beginning download of 995.0 kbytes
2020-05-31 15:12:47 +0000 Notice tlu_ipctask[48950]	Beginning download and installation of packages
2020-05-31 15:12:47 +0000 Notice tlu_ipctask[48950]	Installing bookshelf
2020-05-31 15:12:47 +0000 Warning tlu_ipctask[48950]	running mktexlsr ...
2020-05-31 15:12:47 +0000 Notice tlu_ipctask[48950]	Installation complete; reconfiguring TeX Live
2020-05-31 15:12:52 +0000 Warning tlu_ipctask[48950]	done running mktexlsr.
2020-05-31 15:12:52 +0000 Warning tlu_ipctask[48950]	running mtxrun --generate ...
2020-05-31 15:13:00 +0000 Warning tlu_ipctask[48950]	done running mtxrun --generate.
2020-05-31 15:13:01 +0000 Warning tlu_ipctask[48950]	tlmgr: package log updated: /usr/local/texlive/2020/texmf-var/web2c/tlmgr.log
2020-05-31 15:13:01 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	kqueue noted that pid 48951 exited (/Library/TeX/texbin/tlmgr --machine-readable --repository http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet --persistent-downloads install bookshelf)
2020-05-31 15:13:01 +0000 Notice tlu_ipctask[48950]	child process pid = 48951 exited
2020-05-31 15:13:02 +0000 Notice tlu_ipctask[48950]	exit status of pid = 48951 was 0
2020-05-31 15:13:02 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	waitpid returned 48950, WIFEXITED(0) = 1, errno = 0 (No error)
2020-05-31 15:13:02 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	kqueue noted that tlu_ipctask (pid = 48950) exited with status 0
2020-05-31 15:13:03 +0000 Notice -[TLMMainWindowController _runUpdmapIfNeeded][48174]	/usr/local/texlive/2020/texmf-dist/web2c/updmap.cfg is not in (
    "/Users/josesoeirocarvalho/Library/texmf"
); ignoring
2020-05-31 15:13:03 +0000 Notice -[TLMMainWindowController _refreshFullPackageListFromLocation:offline:][48174]	Refreshing list of all packages…
2020-05-31 15:13:07 +0000 Notice -[TLMLoadDatabaseOperation _dumpDatabaseAtURL:asPropertyList:][48174]	Dump error: finish load http://ftp.eq.uc.pt/software/TeX/systems/texlive/tlnet

2020-05-31 15:13:08 +0000 Notice -[TLMLoadDatabaseOperation main][48174]	Unable to read mirror from tlpdb property list with keys (
    packages
)
2020-05-31 15:13:12 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Reloading local tlpdb from file:///usr/local/texlive/2020/tlpkg/texlive.tlpdb
2020-05-31 15:13:15 +0000 Notice -[TLMDatabase _reloadFromLocalFile][48174]	Took 2.67 seconds to reload local tlpdb
2020-05-31 15:13:16 +0000 Notice +[TLMDatabase packageNodesByMergingLocalWithMirror:][48174]	7158 packages in repository database, 4025 packages in local database
2020-05-31 15:13:19 +0000 Notice -[TLMInfoController showInfoForPackage:location:][48174]	bookshelf found in database; bypassing tlmgr.
2020-05-31 15:20:06 +0000 Notice -[TLMMainWindowController removePackagesWithNames:force:][48174]	Beginning removal of
(
    bookshelf
)
2020-05-31 15:20:06 +0000 Notice -[TLMAuthorizedOperation main][48174]	Checking code signature before running tlu_ipctask as root…
2020-05-31 15:20:06 +0000 Notice /usr/bin/codesign[48174]	/private/var/folders/__/26q5tl3x7lg5pmt_613qx6r40000gn/T/AppTranslocation/2BAEA08A-9B5B-4CE0-94C3-0909859DDC2B/d/TeX Live Utility.app/Contents/MacOS/tlu_ipctask: valid on disk
/private/var/folders/__/26q5tl3x7lg5pmt_613qx6r40000gn/T/AppTranslocation/2BAEA08A-9B5B-4CE0-94C3-0909859DDC2B/d/TeX Live Utility.app/Contents/MacOS/tlu_ipctask: satisfies its Designated Requirement
2020-05-31 15:20:06 +0000 Notice -[TLMAuthorizedOperation main][48174]	Signature was valid, okay to run tlu_ipctask
2020-05-31 15:20:15 +0000 Notice -[TLMAuthorizedOperation main][48174]	Invoking privileged task via AuthorizationExecuteWithPrivileges
2020-05-31 15:20:15 +0000 Notice tlu_ipctask[49228]	dropped privileges to user nobody
2020-05-31 15:20:16 +0000 Notice -[TLMAuthorizedOperation setWrapperPID:][48174]	tlu_ipctask checking in:  tlu_ipctask pid = 49228
2020-05-31 15:20:16 +0000 Notice -[TLMAuthorizedOperation setUnderlyingPID:][48174]	tlu_ipctask checking in: pid = 49229 (/Library/TeX/texbin/tlmgr remove bookshelf)
2020-05-31 15:20:16 +0000 Notice tlu_ipctask[49228]	tlu_ipctask: child HOME = '/var/root'
2020-05-31 15:20:16 +0000 Notice tlu_ipctask[49228]	tlu_ipctask: current HOME = '/var/empty'
2020-05-31 15:20:18 +0000 Notice tlu_ipctask[49228]	tlmgr: saving backups to /usr/local/texlive/2020/tlpkg/backups
2020-05-31 15:20:18 +0000 Notice tlu_ipctask[49228]	[1/0, ??:??/??:??] remove: bookshelf
2020-05-31 15:20:18 +0000 Notice tlu_ipctask[49228]	tlmgr: ultimately removed these packages: bookshelf
2020-05-31 15:20:18 +0000 Notice tlu_ipctask[49228]	running mktexlsr ...
2020-05-31 15:20:20 +0000 Notice tlu_ipctask[49228]	done running mktexlsr.
2020-05-31 15:20:20 +0000 Notice tlu_ipctask[49228]	running mtxrun --generate ...
2020-05-31 15:20:27 +0000 Notice tlu_ipctask[49228]	done running mtxrun --generate.
2020-05-31 15:20:28 +0000 Notice tlu_ipctask[49228]	tlmgr: package log updated: /usr/local/texlive/2020/texmf-var/web2c/tlmgr.log
2020-05-31 15:20:28 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	kqueue noted that pid 49229 exited (/Library/TeX/texbin/tlmgr remove bookshelf)
2020-05-31 15:20:28 +0000 Notice tlu_ipctask[49228]	child process pid = 49229 exited
2020-05-31 15:20:29 +0000 Notice tlu_ipctask[49228]	exit status of pid = 49229 was 0
2020-05-31 15:20:29 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	waitpid returned 49228, WIFEXITED(0) = 1, errno = 0 (No error)
2020-05-31 15:20:29 +0000 Notice -[TLMAuthorizedOperation _runUntilChildExit][48174]	kqueue noted that tlu_ipctask (pid = 49228) exited with status 0
2020-05-31 15:20:29 +0000 Notice -[TLMRemoveOperation main][48174]	ERROR: failed to remove packages {(
    bookshelf
)} (requested removal of (
    bookshelf
))
2020-05-31 15:20:34 +0000 Notice -[TLMLogWindowController awakeFromNib][48174]	Loaded log window controller

**Comment by jmsoeiro** [comment]
Strange, if you list "bookshelf" from the Updates tab, it reports "Not installed", but on the Packages tab, it is marked "Installed".

After reloading tlu, both tabs report "Not installed".

**Comment by jmsoeiro** [comment]
Sorry, not meaning to close the issue.

**Comment by amaxwell** [comment]
Okay, it should really be fixed this time. I had to install Texdoc and documentation to test it properly, since I only have BasicTeX installed. https://github.com/amaxwell/tlutility/releases/download/1.37b3/TeX.Live.Utility.app-1.37b3.dmg

**Comment by jmsoeiro** [comment]
1. I had a really young texlive 2020 full install;
2. The "Not installed" packages looked like new post-2020 packages, available for the first update;
4. When, at last, I did the update, the erroneously documented packages, like *bookshelf* , were associated with the correct documents.
3. I tried console `sudo tlmgr remove bookshelf`, which gave an uninstall error because tlmgr refuses to remove anything out of the "full scheme";
3. but I had been able to install and remove *bookshelf*  **before updating**, although with some strange behavior!
5. At present, I was unable to test anything; and I decided to wipe it off and install BasicTex.
6. Now, when I open "Not installed" *bookshelf*, the package short description is correct and there are no documents associated, which is correct.

I would say that the trouble comes from tlmgr "full scheme" definition.
